project/newEvaluation.html

```diff
@@ -0,0 +1,474 @@
+                    <option value="3">3</option>
+                  </select>
+                  <label for="style">Styling</label>
+                  <select name="style" id="style">
+                    <option value="0">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                  </select>
+                  <label for="dynamic">Dynamic Stylesheets</label>
+                  <select name="dynamic" id="dynamic">
+                    <option value="0">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                  </select>
+                </li>
+          </ul>
+      </fieldset>
+
+      <fieldset>
+        <legend>Javascript</legend>
+        <ul>
+          <li>
+              <label for="data">Data types and variables</label>
+                <select name="data" id="data">
+                  <option value="0">0</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                </select>
+              <label for="object">Object manipulation</label>
+                <select name="object" id="object">
+                  <option value="0">0</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                </select>
+              <label for="dom">DOM manipulation</label>
+                <select name="dom" id="dom">
+                  <option value="0">0</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                </select>
+            </li>
+            <li>
+              <label for="funct">Functions</label>
+                <select name="funct" id="funct">
+                  <option value="0">0</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                </select>
+              <label for="temp">Templating</label>
+                <select name="temp" id="temp">
+                  <option value="0">0</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                </select>
+              <label for="eventh">Event Handling</label>
+              <select name="eventh" id="eventh">
+                <option value="0">0</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                </select>
+            </li>
+            <li>
+              <label for="prot">Prototype & OOP</label>
```

Comment:
Uuuu, yet another one of these strange `&` characters on red background. What on Earth could be up with these? :)